src/html/icons_svg/item-mail__svg.html

```diff
@@ -0,0 +1,4 @@
+<svg class="item-mail__svg" viewBox="0 0 15 11">
```

Comment:
why "item" but not "icon"?

I don't think you need "svg" suffix

check all the places

Reply:
Done.